### PR TITLE
gh-126911: Update credits output

### PR DESCRIPTION
**Pull request by StanFromIreland**
Trivial change updating name of Python Foundation. 

* Issue: gh-126911

### CODE REVIEW

**Comment by hugovk** [comment]
Thanks for the suggestion, but I don't think we should erase their contribution and credit because they had dot com in their name, and the corresponding website is no longer up.

BeOpen.com also has history relating to licensing:

https://docs.python.org/3/license.html

> Trivial change updating name of Python Foundation.

See the history page above, the PSF isn't the same as BeOpen.com with a new name.

**Comment by StanFromIreland** [comment]
@hugovk 

My mistake I have updated my commit to include BeOpen.

File: Misc/NEWS.d/next/Windows/2024-11-16-22-08-41.gh-issue-126911.HchCZZ.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Updated credits command output
```
Comment (terryjreedy) [inline Misc/NEWS.d/next/Windows/2024-11-16-22-08-41.gh-issue-126911.HchCZZ.rst]:
```suggestion
Update credits command output.
```

File: Lib/site.py [hunk 2]
```diff
@@ -442,7 +442,7 @@ def setcopyright():
     """Set 'copyright' and 'credits' in builtins"""
     builtins.copyright = _sitebuiltins._Printer("copyright", sys.copyright)
     builtins.credits = _sitebuiltins._Printer("credits", """\
-    Thanks to CWI, CNRI, BeOpen.com, Zope Corporation and a cast of thousands
+    Thanks to CWI, CNRI, BeOpen, Zope Corporation, the Python Software Foundation and a cast of thousands
     for supporting Python development.  See www.python.org for more information.""")
```
Comment (terryjreedy) [inline Lib/site.py]: Reflow to 80 columns.
```suggestion
    Thanks to CWI, CNRI, BeOpen, Zope Corporation, the Python Software
    Foundation, and a cast of thousands  for supporting Python
    development.  See www.python.org for more information.""")
```

**Review by terryjreedy — CHANGES_REQUESTED** [review]
Approve with 2 tweaks.

**Review by terryjreedy — APPROVED** [review]

**Comment by terryjreedy** [comment]
Hugo: I am not sure whether changing 'BeOpen.com' to 'BeOpen' is proper or whether '.com' is part of the name, as opposes to url.  Even if '.com' was part of the name, the fact that 'BeOpen.com' in the browser resolves to now unrelated 'beopen.com' suggest that deleteing '.com' is a good idea.  I am letting you decide and merge.

Stan: the commit message one adds is intended to be put above, not below, the boilerplate comment.  Much easier to edit when above.

File: Lib/site.py [hunk 3]
```diff
@@ -442,8 +442,9 @@ def setcopyright():
     """Set 'copyright' and 'credits' in builtins"""
     builtins.copyright = _sitebuiltins._Printer("copyright", sys.copyright)
     builtins.credits = _sitebuiltins._Printer("credits", """\
-    Thanks to CWI, CNRI, BeOpen.com, Zope Corporation and a cast of thousands
-    for supporting Python development.  See www.python.org for more information.""")
+    Thanks to CWI, CNRI, BeOpen, Zope Corporation, the Python Software
+    Foundation, and a cast of thousands  for supporting Python
```
Comment (encukou) [inline Lib/site.py]:
```suggestion
    Foundation, and a cast of thousands for supporting Python
```

**Review by hugovk — APPROVED** [review]
As far as I can tell, "BeOpen.com" was the full name. It was from the dotcom bubble days, and they've long been out of business (as is visible from the new owners of the domain).

We should keep "BeOpen.com" for the [copyright](https://docs.python.org/3/copyright.html) and [licence](https://docs.python.org/3/license.html), but I thinks it's fine to use a shorter version here in the REPL credits.

**Comment by hugovk** [comment]
@StanFromIreland Tip for next time, we prefer not to rebase branches and force push in this repo, thanks!

**Pull request merged**